naff/ext/debug_extension/utils.py

```diff
@@ -164,3 +165,30 @@ def make_table(rows: list[list[Any]], labels: Optional[list[Any]] = None, center
         lines.append(_make_data_line(column_widths, row, data_left, data_middle, data_right, align))
     lines.append(_make_solid_line(column_widths, "╰", "┴", "╯"))
     return "\n".join(lines)
+
+
+def get_all_commands(module: ModuleType) -> Dict[str, Callable]:
```

Comment:
Why is this in this file?

Reply:
I couldn't think of a better place to put it at the time. It's a useful utility regardless of being used in jurigged or not, and when I added it it was next to the jurigged extension.

This could be moved to `naff.utils` if you think that's a better spot

Reply:
Yeah, that feels better.

Reply:
Done in latest commit, it's now in `naff.client.utils.misc_utils` with other search-like functions